src/parameterizations/lateral/MOM_lateral_mixing_coeffs.F90

```diff
@@ -1036,6 +1054,8 @@ subroutine calc_slope_functions_using_just_e(h, G, GV, US, CS, e)
   ! and midlatitude deformation radii, using calc_resoln_function as a template.
 
   !$OMP parallel do default(shared) private(E_x,E_y,S2,Hdn,Hup,H_geom,N2)
+
```

Comment:
Please remove this added white space and the other white space changes in `calc_slope_functions_using_just_e()`, at lines 1009 of the original code and lines 1085, 1109, 1113, 1132 and 1137 of the revised code. This PR is no longer making any substantive changes in `calc_slope_functions_using_just_e()`, so there is no need for these white-space changes either.  

(This is very minor, and I would not suggest this were there not already the need for other changes to fix likely problems with this PR.)